packages/cozy-harvest-lib/src/components/cards/LaunchTriggerAlert.jsx

```diff
@@ -1,57 +1,83 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
 
 import Alert from 'cozy-ui/transpiled/react/Alert'
 import Button from 'cozy-ui/transpiled/react/Buttons'
 import Typography from 'cozy-ui/transpiled/react/Typography'
 import Spinner from 'cozy-ui/transpiled/react/Spinner'
+import Snackbar from 'cozy-ui/transpiled/react/Snackbar'
 
-import KonnectorIcon from '../KonnectorIcon'
 import { getLastSuccessDate, getKonnectorSlug } from '../../helpers/triggers'
 import { isRunnable } from '../../helpers/konnectors'
 import { useFlowState } from '../../models/withConnectionFlow'
+import { SUCCESS } from '../../models/flowEvents'
+import KonnectorIcon from '../KonnectorIcon'
 import { makeLabel } from './helpers'
 
 export const LaunchTriggerAlert = ({ flow, f, t, disabled }) => {
-  const { trigger, running, expectingTriggerLaunch } = useFlowState(flow)
+  const [showSuccessSnackbar, setShowSuccessSnackbar] = useState(false)
+  const { trigger, running, expectingTriggerLaunch, status } =
+    useFlowState(flow)
   const { launch, konnector } = flow
+
   const lastSuccessDate = getLastSuccessDate(trigger)
   const isKonnectorRunnable = isRunnable({ win: window, konnector })
+  const isSuccess = status === SUCCESS
+
+  const hideSuccessSnackbar = () => setShowSuccessSnackbar(false)
```

Comment:
what is the benefit to create a new method? Instead of calling directly the setter?

Reply:
d'éviter d'écrire des handler inline `<Snackbar open={showSuccessSnackbar} onClose={() => setShowSuccessSnackbar(false)}>` on s'est dit en front que ce serait globalement un peu moins couteux en RAM

Reply:
C'est exactement la même chose en RAM entre: 

```
const showSuccessSnackbar = () => setS...()
<Snackbar open={showSuccessSnackbar} />
```

et 
```
<Snackbar open={() => setS...()} />
```

Dans tous les cas, à chaque re-render tu vas recréer la méthode et donc ajouter ça en RAM. Et je dirais même que le fait de stocker la fonction dans la var fait une assignation supplémentaire et donc consommera plus de RAM. 
Le seul moyen d'éviter ça, c'est le useCallback.